scalableminds/webknossos
# Respect packing degree of segmentation layer in shader

### URL of deployed dev instance (used for testing):
- `http://fixuint8seg.webknossos.xyz` (cannot login even after several re-deployments :() [Works now :)]

### Steps to test:
- load a 32 bit, 16 bit as well as 8 bit segmentation and ensure rendering is correct

------
